sdks/java/io/google-cloud-platform/src/test/java/org/apache/beam/sdk/io/gcp/bigquery/BigQueryIOWriteTest.java

```diff
@@ -1593,6 +1595,91 @@ public void testStreamingStorageApiWriteWithAutoShardingWithErrorHandling() thro
     storageWriteWithErrorHandling(true);
   }
 
+  private void storageWriteWithSuccessHandling(boolean columnSubset) throws Exception {
```

Comment:
nit: I believe this doesn't test the batch code path because TestStream is always used. It's probably fine though since the real filtering logic is not based there

Reply:
I don't think so - if useStreaming==false, we call Create.of(elements) instead of using TestStream

Reply:
Ahh I skipped over that line. Looks good